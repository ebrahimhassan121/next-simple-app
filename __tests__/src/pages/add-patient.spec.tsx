import { render } from '@testing-library/react';
import AddPatient from '@/pages/add-patient';

describe(`@/components/pages/add-patient`, () => {
  it(`should render add patient page correctly`, () => {
    const { container } = render(<AddPatient />);
    expect(container).toMatchSnapshot();
  });
});
