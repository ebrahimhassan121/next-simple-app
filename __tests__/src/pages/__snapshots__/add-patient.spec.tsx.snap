// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@/components/pages/add-patient should render add patient page correctly 1`] = `
<div>
  <div
    class="MuiContainer-root makeStyles-container-1 MuiContainer-maxWidthLg"
  >
    <div
      class="MuiPaper-root makeStyles-headerContainer-2 MuiPaper-elevation1 MuiPaper-rounded"
    >
      <h4
        class="MuiTypography-root MuiTypography-h4"
      >
        Add Patient
      </h4>
      <a
        aria-disabled="false"
        class="MuiButtonBase-root MuiIconButton-root"
        href="/"
        tabindex="0"
      >
        <span
          class="MuiIconButton-label"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
            />
          </svg>
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </a>
    </div>
    <form>
      <div
        class="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root-5 MuiFormControl-fullWidth"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled"
          data-shrink="false"
          for="name"
          id="name-label"
        >
          NAME
        </label>
        <div
          class="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-fullWidth MuiInputBase-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiFilledInput-input"
            id="name"
            type="text"
            value=""
          />
        </div>
      </div>
      <div
        class="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root-5 MuiFormControl-fullWidth"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled"
          data-shrink="false"
          for="age"
          id="age-label"
        >
          AGE
        </label>
        <div
          class="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-fullWidth MuiInputBase-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiFilledInput-input"
            id="age"
            type="text"
            value=""
          />
        </div>
      </div>
      <div
        class="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root-5 MuiFormControl-fullWidth"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled"
          data-shrink="false"
          for="phone"
          id="phone-label"
        >
          PHONE
        </label>
        <div
          class="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-fullWidth MuiInputBase-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiFilledInput-input"
            id="phone"
            type="text"
            value=""
          />
        </div>
      </div>
      <div
        class="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root-5 MuiFormControl-fullWidth"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled"
          data-shrink="false"
          for="address"
          id="address-label"
        >
          ADDRESS
        </label>
        <div
          class="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-fullWidth MuiInputBase-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiFilledInput-input"
            id="address"
            type="text"
            value=""
          />
        </div>
      </div>
      <div
        class="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root-5 MuiFormControl-fullWidth"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled"
          data-shrink="false"
          for="healthInfo"
          id="healthInfo-label"
        >
          HEALTHINFO
        </label>
        <div
          class="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-multiline MuiFilledInput-multiline"
        >
          <textarea
            aria-invalid="false"
            class="MuiInputBase-input MuiFilledInput-input MuiInputBase-inputMultiline MuiFilledInput-inputMultiline"
            id="healthInfo"
            rows="1"
            style="height: 0px; overflow: hidden;"
          />
          <textarea
            aria-hidden="true"
            class="MuiInputBase-input MuiFilledInput-input MuiInputBase-inputMultiline MuiFilledInput-inputMultiline"
            readonly=""
            style="visibility: hidden; position: absolute; overflow: hidden; height: 0px; top: 0px; left: 0px; transform: translateZ(0); width: 100%;"
            tabindex="-1"
          />
        </div>
      </div>
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-fullWidth"
        tabindex="0"
        type="submit"
      >
        <span
          class="MuiButton-label"
        >
          Add patient
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </form>
  </div>
</div>
`;
